src/windows/FileTransferProxy.js

```diff
@@ -49,6 +49,17 @@ function nativePathToCordova(path) {
     return String(path).replace(/\\/g, '/');
 }
 
+function alreadyCancelled(opId, errorCb, filePath, server) {
+    var op = fileTransferOps[opId];
+    if (op && op.state === FileTransferOperation.CANCELLED) {
+        // Here we should call errorCB with ABORT_ERR error
+        errorCb(new FTErr(FTErr.ABORT_ERR, nativePathToCordova(filePath), server));
```

Comment:
This side effect should not be part of the function. Please place it outside, into the calling code. It will be duplicated, but that is still somewhat OK; it's definitely better than someone calling if `alreadyCancelled()` twice in a row and getting their callback called twice.